src/sections/Community/Handbook/projects.js

```diff
@@ -16,19 +19,38 @@ const Maintainer= () => {
+                <li><h4><a href="https://layer5.io/projects/image-hub">Image-Hub :</a></h4> </li>
+                <p>Image Hub is a sample application written to run on Consul for exploring WebAssembly modules used as Envoy filters are written in Rust. These modules can be used to implement multi-tenancy or to implement per-user rate-limiting in your application's endpoints. This application was first demonstrated at DockerCon 2020. <br/><a href="https://github.com/layer5io/image-hub"><img className="channels-img" src={github} alt="github"/></a> </p>
+                <li><h4><a href="https://layer5.io/projects/getnighthawk">GetNightHawk :</a></h4> </li>
+                <p>This Is a Layer 7 (HTTP/HTTPS/HTTP2) performance characterization tool. Nighthawk is Envoy’s load generator and is written in C++. Meshery integrates Nighthawk as one of (currently) three choices of load generators for characterizing and managing the performance of service meshes and their workloads. <br/><a href="https://github.com/layer5io/getnighthawk"><img className="channels-img" src={github} alt="github"/></a> </p>
+              </ul>
+            </p>
+            <p><h3><a href="https://layer5.io/service-mesh-management/meshery"><img margin="10px" width="3%" align="center" src={meshery} alt="Meshery"/> Meshery </a></h3></p>
+            <p>Meshery and its components Meshery Operator and MeshSync  
+              <ul>
+                <li><h4><a href="https://layer5.io/service-mesh-management/meshery">Meshery</a></h4> </li>
+                <p>It is a multi-service mesh management plane offering lifecycle management of more types of service meshes than any other tool available today. Meshery facilitates adopting, configuring, operating, and managing the performance of different service meshes and incorporates the collection and display of metrics from applications running on top of any service mesh.<br/> <a href="https://github.com/meshery"><img className="channels-img" src={github} alt="github"/></a> </p>
+                <li><h4><a href="https://meshery.io/">Meshery.io :</a></h4> </li>
+                <p>Website for Meshery, the multi-mesh manager. <br/><a href="https://github.com/meshery/meshery.io"><img className="channels-img" src={github} alt="github"/></a> </p>
+                <li><h4><a href="https://layer5.io/service-mesh-management/meshery/meshery-operator">Meshery-operator :</a></h4> </li>
+                <p>Meshery Operator is the multi-service mesh operator and implementation of MeshSync. <br/><a href="https://github.com/meshery/meshery-operator"><img className="channels-img" src={github} alt="github"/></a> </p>
+              </ul>
+            </p>
+            <h3><a href="https://smp-spec.io/"><img className="channels-img" src={smp} /> Service Mesh Performance</a></h3>
```

Comment:
![smpicon](https://user-images.githubusercontent.com/85789734/134772005-564c2aa7-df34-40b8-a56b-54cb4fee9d93.jpg)
This should be changed to the SMP logo. Check if you're importing from a different address.